packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -60,9 +60,38 @@ const studioLocaleStrings = {
   'timeAgo.seconds.ago_one': '{{count}} minute ago',
   'timeAgo.seconds.ago_other': '{{count}} second ago',
   /* Relative time, granularity: seconds, using a minimal format*/
-  'timeAgo.seconds.minimal': '{{count}m',
+  'timeAgo.seconds.minimal': '{{count}}m',
```

Comment:
@rexxars I made this change just for the sake of fixing the interface.
However, this whole section (how we're doing the time) is something that I strongly suggest we change / minimise when it comes to the number of keys. 
From a DX perspective (the number of things I as a dev would need to write) and from a Business perspective (how expensive I see this becoming) I feel is what makes sense.

I will probably keep it outside of the scope of this PR and move it to its own story but a thought to keep in mind.

Reply:
Yeah, I also think that this is _potentially_ the wrong approach for relative time anyway - there is a `Intl.RelativeTime` API that I think is probably a better candidate. I want to see if we can use that instead of manually defining these strings - so we should create a story for "Investigate best approach for relative time"

Reply:
Created a story for this 👍